code/game/objects/items.dm

```diff
@@ -48,6 +48,8 @@
 	var/zoomdevicename = null //name used for message when binoculars/scope is used
 	var/zoom = 0 //1 if item is actively being used to zoom. For scoped guns and binoculars.
 
+	var/contained_sprite = FALSE //TRUE if item_state, lefthand, righthand, and worn sprite are all in one dmi
```

Comment:
Something wrong with that comment. 
May be that should be `icon_state` instead of `item_state`?
Or "object icon and related mob overlays".

Reply:
Yeah, that would be more specific, it's essentially saying item state four times over. Will resolve.